.github/CODEOWNERS

```diff
@@ -53,6 +53,7 @@
 /fluxcd/                                 @melchiormoulin melchior.moulin@gmail.com
 /gatekeeper/                             @arapulido ara.pulido@datadoghq.com
 /gitea/                                  @FlorentClarret
+/go_pprof_scraper/                       @nsrip-dd
```

Comment:
Could you also update this section so the doc team will keep an eye on the docs? https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L167

Reply:
Done